rclpy/test/test_node.py

```diff
@@ -171,6 +188,72 @@ def test_node_names_and_namespaces(self):
         # test that it doesn't raise
         self.node.get_node_names_and_namespaces()
 
+    def test_get_publishers_subscriptions_info_by_topic(self):
+        topic_name = 'test_topic_info'
+        fq_topic_name = '%s/%s' % (TEST_NAMESPACE, topic_name)
```

Comment:
Any reason not to use python's `format()`?

Reply:
No particular reason, just maintaining consistency across the tests (as other tests use the `%` as well). But, I think str.format() looks cleaner.